object/user.go

```diff
@@ -47,6 +47,7 @@ type User struct {
 	Email             string   `xorm:"varchar(100) index" json:"email"`
 	EmailVerified     bool     `json:"emailVerified"`
 	Phone             string   `xorm:"varchar(100) index" json:"phone"`
+	PhonePrefix       string   `xorm:"varchar(100)" json:"phonePrefix"`
```

Comment:
https://stackoverflow.com/questions/8284647/how-to-store-phone-numbers-on-mysql-databases
Both phone prefix and number save as a varchar maybe better. Formatting should be done in the view.

Reply:
@leo220yuyaodog one answer sugggested storing country code separately

![image](https://user-images.githubusercontent.com/3787410/217162456-d6ddfa31-805f-4b4a-aebf-f0b8c7107025.png)